source/_patterns/00-atoms/02-typography/09-anker.md

```diff
@@ -0,0 +1,4 @@
+---
+title: Anker
```

Comment:
I think you mean "anchor", the html element, isn't it?

Reply:
In 4 other places too.
Done